### PR TITLE
Adding attributes for svg paths.

### PR DESCRIPTION
**Pull request by HejdesGit**
Would like to use this module but I'm using other design on our paths eg. 'stroke-linecap': 'round', 'stroke-linejoin': 'round' on the paths. I made it possible to add attributes to the SVG paths.

Take a look and see if you like it. Or just hit me back if something should be fixed.

There are also some problems with the formatting of your code. I'm using Webstorm and there seems to be a lot of extra tabs and spaces.
Have you considered using: [EditorConfig](http://editorconfig.org/)?

### CODE REVIEW

File: example/index.html [hunk 1]
```diff
@@ -59,6 +59,7 @@
           float: right;
           padding: 10px;
         }
+      }
```
Comment (HejdesGit) [inline example/index.html]: There was missing an brace for the: @media only screen and (min-device-width: 768px) {

**Comment by HejdesGit** [comment]
I'm setting the attribute via css props instead. Thought there was a use case for setting attributes but on second thought, maybe not.

**Comment by naikus** [comment]
Thanks for pointing about EditorConfig. And I too think it'd be a good idea to style via CSS. Btw I'm going to fix the example file as suggested by you. Thanks!

**Comment by naikus** [comment]
Or you can submit a pull request for that :)

**Comment by HejdesGit** [comment]
I just did. But I closed it since I didn't see the use case right now :)

Another thing. Is there a reason for the viewbox to be that big? 1000 1000? SVG is not my expertise, just wondering. Because it looks like it paints the SVG rather big then shrinks it down with the container class.

**Pull request closed without merging**